cli/app.go

```diff
@@ -465,6 +465,18 @@ var app = &cli.App{
 						},
 					},
 				},
+				{
+					Name:  "list-oauth-apps",
```

Comment:
do we want this to be the command? or should it be something like `organization oauth list` so we can also have `organization oauth create` etc. 

would check scope and check with @maxhorowitz